src/thunderbots/software/ai/hl/stp/evaluation/indirect_chip.h

```diff
@@ -0,0 +1,121 @@
+#include "ai/world/world.h"
+#include "geom/angle.h"
+#include "geom/point.h"
+#include "util/parameter/dynamic_parameters.h"
+
+/**
+ * The indirect chip and chase target evaluation function returns a target point for the
+ * chipper and chaser.
+ * Other related evaluation functions are also included in this file.
+ */
+namespace Evaluation
+{
+    typedef Point Vector2;
+    template <size_t N>
```

Comment:
These typedefs are already defined in `geom/util.h`, so they can probably be removed from here.

The `typedef Point Vector2;` also exists in the `geom/point.h` file (except it's just `Vector` instead of `Vector2`)

Reply:
Line has been removed.

Reply:
Doesn't look like this line got changed? Just to be totally clear, we shouldn't need to replace these lines, we should just be able to use `Vector`, `Poly`, `Triangle` if we add `#include "geom/util.h" to the top of this file.

Reply:
Ah, you're right! The four lines have been removed and changed `Triangle` -> `LegacyTriangle` after Jordan's changes in #480 .